src/java/org/apache/cassandra/audit/AuditLogManager.java

```diff
@@ -114,6 +113,11 @@ private void log(AuditLogEntry logEntry)
     {
         if (!filter.isFiltered(logEntry))
         {
+            // we may need to obfuscate password (CASSANDRA-16669)
```

Comment:
@sumanth-pasupuleti I believe this is not acually necessary, the code is pretty much self-evident.